Exception/Exception.php

```
@@ -1,28 +0,0 @@
-<?php
```

I want to keep this class: if i need to throw an exception who is not a ConnectionException, i will use this one

---

Maybe better to create an Interface Exception